packages/hydrogen-react/src/analytics.test.ts

```diff
@@ -331,9 +331,9 @@ describe('analytics', () => {
       };
       // @ts-ignore
       global.PerformanceNavigation = {
-        TYPE_NAVIGATE: 1,
-        TYPE_RELOAD: 2,
-        TYPE_BACK_FORWARD: 3,
```

Comment:
Don't change these values - These are code mapping for `PerformaneNavigation` api

https://developer.mozilla.org/en-US/docs/Web/API/PerformanceNavigation#instance_properties

Reply:
Hmm... but TS complains these are not the correct values... and that MDN doc you link also supports 0,1,2 instead of 1,2,3? Or am I missing something obvious?

<img width="797" alt="image" src="https://github.com/Shopify/hydrogen/assets/1634092/0530ed51-4e7c-4291-9298-93c7327805f9">


---

This is just a unit test so I guess I can just ts-ignore my way but even MDN says it should start with 0, no? 🤔